ush/set_rrfs_config.sh

```diff
@@ -54,7 +54,7 @@ if [[ $DO_RETRO == "TRUE" ]] ; then
       EXTRN_MDL_SOURCE_BASEDIR_ICS=${RETRODATAPATH}/GEFS
     elif [[ ${EXTRN_MDL_NAME_ICS} == "GDASENKF" ]]; then
         EXTRN_MDL_SOURCE_BASEDIR_ICS="${RETRODATAPATH}/enkf/atm"
-    if
+    fi
```

Comment:
This was fixed in Ming's PR which was merged a few minutes ago. 

Could you do the following to update your hercules branch? 
```
git pull origin hercules
git push -u shawn hercules
```
Thanks!

Reply:
Done. Thanks for pointing this out!